src/overlays/actors/ovl_En_Demo_heishi/z_en_demo_heishi.h

```diff
@@ -7,12 +7,46 @@ struct EnDemoheishi;
 
 typedef void (*EnDemoheishiActionFunc)(struct EnDemoheishi*, GlobalContext*);
 
+typedef enum {
+    /* 0 */ DEMOHEISHI_LIMB_NONE,
+    /* 1 */ DEMOHEISHI_LIMB_ROOT,
+    /* 2 */ DEMOHEISHI_LIMB_PELVIS,
+    /* 3 */ DEMOHEISHI_LIMB_LEFT_THIGH,
+    /* 4 */ DEMOHEISHI_LIMB_LEFT_CALF,
+    /* 5 */ DEMOHEISHI_LIMB_LEFT_FOOT,
+    /* 6 */ DEMOHEISHI_LIMB_RIGHT_THIGH,
+    /* 7 */ DEMOHEISHI_LIMB_RIGHT_CALF,
+    /* 8 */ DEMOHEISHI_LIMB_RIGHT_FOOT,
+    /* 9 */ DEMOHEISHI_LIMB_TORSO,
+    /* 10 */ DEMOHEISHI_LIMB_LEFT_SHOULDER,
+    /* 11 */ DEMOHEISHI_LIMB_LEFT_FOREARM,
+    /* 12 */ DEMOHEISHI_LIMB_LEFT_HAND,
+    /* 13 */ DEMOHEISHI_LIMB_RIGHT_SHOULDER,
+    /* 14 */ DEMOHEISHI_LIMB_RIGHT_FOREARM,
+    /* 15 */ DEMOHEISHI_LIMB_RIGHT_HAND_WITH_SPEAR,
+    /* 16 */ DEMOHEISHI_LIMB_HEAD,
+    /* 17 */ DEMOHEISHI_LIMB_MAX
+} EnDemoheishiLimbs;
```

Comment:
And on the "extremely minor formatting nitpicks" list,
```suggestion
    /*  0 */ DEMOHEISHI_LIMB_NONE,
    /*  1 */ DEMOHEISHI_LIMB_ROOT,
    /*  2 */ DEMOHEISHI_LIMB_PELVIS,
    /*  3 */ DEMOHEISHI_LIMB_LEFT_THIGH,
    /*  4 */ DEMOHEISHI_LIMB_LEFT_CALF,
    /*  5 */ DEMOHEISHI_LIMB_LEFT_FOOT,
    /*  6 */ DEMOHEISHI_LIMB_RIGHT_THIGH,
    /*  7 */ DEMOHEISHI_LIMB_RIGHT_CALF,
    /*  8 */ DEMOHEISHI_LIMB_RIGHT_FOOT,
    /*  9 */ DEMOHEISHI_LIMB_TORSO,
    /* 10 */ DEMOHEISHI_LIMB_LEFT_SHOULDER,
    /* 11 */ DEMOHEISHI_LIMB_LEFT_FOREARM,
    /* 12 */ DEMOHEISHI_LIMB_LEFT_HAND,
    /* 13 */ DEMOHEISHI_LIMB_RIGHT_SHOULDER,
    /* 14 */ DEMOHEISHI_LIMB_RIGHT_FOREARM,
    /* 15 */ DEMOHEISHI_LIMB_RIGHT_HAND_WITH_SPEAR,
    /* 16 */ DEMOHEISHI_LIMB_HEAD,
    /* 17 */ DEMOHEISHI_LIMB_MAX
} EnDemoheishiLimbs;
```